controllers/storagecluster/backingstorageclasses.go

```diff
@@ -29,41 +30,49 @@ func (obj *backingStorageClasses) ensureCreated(r *StorageClusterReconciler, sc
 	if err != nil {
 		return reconcile.Result{}, err
 	}
-	existingBackingStorageClasses, err := getBackingStorageClasses(r)
+	existingBackingStorageClasses := &v1.StorageClassList{}
+	err = r.Client.List(r.ctx,
+		existingBackingStorageClasses,
+		client.MatchingLabels(map[string]string{
+			backingStorageClassLabel: strconv.FormatBool(true),
+		}),
+	)
```

Comment:
If we are doing a multi-line func call formatting, make sure that each argument has its own line:
``` suggestion 
	err = r.Client.List(
		r.ctx,
		existingBackingStorageClasses,
		client.MatchingLabels(map[string]string{
			backingStorageClassLabel: strconv.FormatBool(true),
		}),
	)
```